packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-node-ciemss.vue

```diff
@@ -247,13 +281,13 @@ watch(
 	() => props.node.state.inProgressForecastId,
 	async (id) => {
 		if (!id || id === '') return;
-		// Check simulation status and update the state
-		const simResponse: Simulation | null = await getSimulation(id);
-		if (simResponse?.status !== ProgressState.Complete) return;
-		const state = _.cloneDeep(props.node.state);
-		state.inProgressForecastId = '';
-		state.forecastId = id;
-		emit('update-state', state);
+		const response = await pollResult(id);
+		if (response.state === PollerState.Done) {
+			const state = _.cloneDeep(props.node.state);
+			state.inProgressForecastId = '';
+			state.forecastId = id;
+			emit('update-state', state);
+		}
```

Comment:
I think there may be some misunderstanding on what the task is asking, let me try to elaborate here.

As briefly mentioned before, the push-event had been broken for a significant amount of time, so we wanted to switch over to do polling since we know it works and is consistent across other operators. For the simulate operator, it would either be using polling or push-event, but not both (as it is doing currently) - as you would be kind of double-counting or double-executing in some sense with both mechanisms running simultaneously.

Moreover, there are two items we are polling for:
- A base simulation, that is a simulation without an intervention policy, and
- A simulation with an intervention policy
These are the "baseForecastId", "forecastId" ... fields in the operator's state. 
So I think this is only half working, but may appear to work because:
- you may not have exercised the intervention use case, and/or
- we had addressed a couple of problems with push-events in the meantime

To have a simulate with intervention, you need to create an intervention-policy operator and pipe it into the simulate-operator. Feel free to reach out if more clarification is needed here.